### ISSUE TITLE
Install fails on Bullseye because libx264 / libvpx changed names

### ISSUE THREAD

**Issue by alexAubin** [YunoHost-Apps/jellyfin_ynh#77]
```
13900 INFO DEBUG -  jellyfin-ffmpeg : Depends: libvpx5 (>= 1.6.0) but it is not installable
13900 INFO DEBUG -                    Depends: libx264-155 but it is not installable
13900 INFO DEBUG -                    Depends: libx265-165 (>= 2.9) but it is not installable
```

Apparently libvpx5 is now libvpx6 https://packages.debian.org/bullseye/libvpx6, and there's a libx264-160 https://packages.debian.org/bullseye/libx264-160, but I don't have much idea if that's a straightforward replacement or there's more to it

**Comment by tituspijean** [comment]
As always I will be extra super duper wary about package management with this app. Is it possible to deploy a testing "buster" branch and test this one on the CI? Or should I just wait for the release and worry about it then?

**Comment by Salamandar** [comment]
Will be fixed by #87 .
